### DISCUSSION TITLE
What's the backward compatibility for this SDK?

### DISCUSSION THREAD

**Discussion by xiaomi7732** [kubernetes-client/csharp#1175] [Q&A]
1. If I am using `K8sClient 10.0`, does it suppose to work with a cluster of version `1.22` (26-4)?

FWIW, in the doc, I can only find info looking forward, for example, do I understand this correctly, K8sClient.8.0 generated from `1.24` works with `1.25` & `1.26`.

SDK Version | Kubernetes Version | .NET Targeting
-- | -- | --
10.0 | 1.26 | net6.0;net7.0;net48*;netstandard2.0*
9.1 | 1.25 | netstandard2.1;net6.0;net7.0;net48*;netstandard2.0*
9.0 | 1.25 | netstandard2.1;net5.0;net6.0;net48*;netstandard2.0*
8.0 | 1.24 | netstandard2.1;net5.0;net6.0;net48*;netstandard2.0*
7.2 | 1.23 | netstandard2.1;net5.0;net6.0;net48*;netstandard2.0*
7.0 | 1.23 | netstandard2.1;net5.0;net6.0

**Comment by tg123** [comment]
it may work with 1.22 even 1.12 but no guarantees.

for example, pod related apis are very stable, while some newer fancy apis are not.
